ui/packages/catalog-ui-search/src/main/webapp/js/widgets/openlayers.line.js

```diff
@@ -97,14 +98,14 @@ Draw.LineView = Marionette.View.extend({
   updatePrimitive(model) {
     const polygon = this.modelToPolygon(model)
     // make sure the current model has width and height before drawing
-    if (polygon && !_.isUndefined(polygon)) {
+    if (polygon !== undefined && !validateGeo('polygon', JSON.stringify(polygon.getCoordinates())).error) {
```

Comment:
❗️ with `polygon` here, the error is true for lines with 3 or less points because `A minimum of 4 points needed for polygon`
```suggestion
    if (polygon !== undefined && !validateGeo('line', JSON.stringify(polygon.getCoordinates())).error) {
```